demos/storybook/.gitignore

```diff
@@ -0,0 +1,65 @@
+# Logs
```

Comment:
I actually don't know if this file will do anything since we have a gitignore file at the root of the project. What's the intent here?

Reply:
I'm not entirely sure why this got here; maybe this was generated when I locally made a firebase project in this folder.  I've removed this file.